lib/cdo/pardot.rb

```diff
@@ -109,7 +109,6 @@ def self.update_pardot_ids_of_new_contacts
       results_in_response = 0
 
       # Process every prospect in the response.
-      id_max = email_to_pardot_id.values.max
```

Comment:
This was a leftover line from @ashercodeorg's refactor that was unintended and needs to be removed.